scrape.py

```diff
@@ -173,15 +174,16 @@ def extract_listings(soup):
             'rent': rent,
             'area': area,
         }
+        print(listings)
```

Comment:
Right now the scraper doesn't output anything to STDOUT, all output goes into the database or into the log file. I'd like to keep it that way for the moment.